aws_lambda_powertools/utilities/data_classes/kafka_event.py

```diff
@@ -69,18 +72,9 @@ def decoded_headers(self) -> Dict[str, bytes]:
 
     def get_header_value(
         self, name: str, default_value: Optional[Any] = None, case_sensitive: bool = True
-    ) -> Optional[bytes]:
+    ) -> Optional[str]:
         """Get a decoded header value by name."""
-        if case_sensitive:
-            return self.decoded_headers.get(name, default_value)
-        name_lower = name.lower()
-
-        return next(
-            # Iterate over the dict and do a case-insensitive key comparison
-            (value for key, value in self.decoded_headers.items() if key.lower() == name_lower),
-            # Default value is returned if no matches was found
-            default_value,
-        )
+        return get_header_value(self.decoded_headers, name, default_value, case_sensitive)
```

Comment:
prefer kwargs to ease reading/refactor signature

Reply:
Make total sense, Heitor!
Fixed